docs/usage/types.md

```diff
@@ -492,6 +492,9 @@ _(This script is complete, it should run "as is")_
 `PostgresDsn`
 : a postgres DSN style URL; see [URLs](#urls)
 
+`RabbitMqDsn`
+: a RabbitMQ DSN style URL; see [URLs](#urls)
```

Comment:
Regard the screenshot of the other PR you asked about, I would just put something like
```suggestion
: an `AMQP` DSN style URL as used by RabitMQ, StormMQ, ActiveMQ etc.; see [URLs](#urls)
```

Reply:
Ah. :ok_hand: